config-model/src/test/java/com/yahoo/vespa/model/content/DistributorTest.java

```diff
@@ -269,6 +269,7 @@ public void testPortOverride() {
 
         cluster.getChildren().get("0").getConfig(builder);
         StorCommunicationmanagerConfig config = new StorCommunicationmanagerConfig(builder);
+        assertTrue(config.mbus().dispatch_on_encode());
```

Comment:
Consider pulling this out into a separate test since it's not really port override related (though networking related)

Reply:
Done